hubble-relay.Dockerfile

```diff
@@ -17,6 +17,8 @@ RUN go get -d github.com/google/gops && \
     strip /go/bin/gops
 
 FROM scratch
+ARG CILIUM_SHA=""
```

Comment:
Please add these also to every intermediate image we are using (every `FROM` Dockerfile command creates a new image), so they need to be added on current lines 2, 8 and 11.